lib/filesystem_parser.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+class LsblkParser
```

Comment:
Specs for all these that show the expected input to be parsed would be nice.

Reply:
Specs are added for all the parsers.